### PR TITLE
Loginstyling

### PR DESCRIPTION
**Pull request by Kmmontes**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -26,13 +26,14 @@ class App extends Component {
       <Router>
       <Header {...this.props} />
       <Switch>
-        <Route exact path="/" component={Home} />
+        <Route exact path="/"> <Home /> </Route>
 
-        <Route path="/gooseynew" component={GooseyNew} />
 
-        <Route path="/gooseyedit" component={GooseyEdit} />
+        {/* <Route path="/gooseynew"> <GooseyNew />  </Route> */}
```
Comment (elysemontano) [inline app/javascript/components/App.js:32]: Why are these commented out?

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -13,35 +16,150 @@ class Header extends Component {
...
-    )
-  }
+// begin navbar
+
+//     <div>
+//     <div>
+//   <Navbar
+//    color="dark" dark expand="md" fixed="top">
+// </Navbar>
+//     </div>
```
Comment (elysemontano) [inline app/javascript/components/components/Header.js:26]: Let's try to remove commented out code

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left a few comments on Header and Home to address.

File: app/javascript/components/App.js (hunk 1 above)
Comment (VanDexter) [reply app/javascript/components/App.js:32]: We are going to uncomment them as we create those pages.

Comment (elysemontano) [reply app/javascript/components/App.js:32]: For future, best practice is to remove commented out code before merging to main.

File: app/javascript/components/components/Header.js [hunk 3]
```diff
@@ -1,47 +1,74 @@
...
+                                <NavItem>
+                                    <NavLink to="/mygooseys">My Gooseys</NavLink>
+                                </NavItem>
+                                <NavItem>
+                                    <NavLink to="/new">Add Goosey</NavLink>
+                                </NavItem>
+                                <NavItem>
+                                    <a href={sign_out_route}>Sign Out</a>
+                                </NavItem>
+                            </>
```
Comment (elysemontano) [inline app/javascript/components/components/Header.js:55]: You may be able to remove the react fragments here.

File: app/javascript/components/pages/GooseyEdit.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class GooseyEdit extends Component {
+  render() {
+    return (
+    <div>
+    <h2>GooseyEdit</h2>
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyEdit.js:7]: This should be indented.

File: app/javascript/components/pages/GooseyIndex.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class GooseyIndex extends Component {
+  render() {
+    return (
+    <div>
+    <h2>GooseyIndex</h2>
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyIndex.js:7]: This should also be indented.

File: app/javascript/components/pages/GooseyNew.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class GooseyNew extends Component {
+  render() {
+    return (
+    <div>
+    <h2>GooseyNew</h2>
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyNew.js:7]: Indentation.

File: app/javascript/components/pages/GooseyShow.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class GooseyShow extends Component {
+  render() {
+    return (
+    <div>
+    <h2>GooseyShow</h2>
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyShow.js:7]: Indentation

File: app/javascript/components/pages/Home.js [hunk 8]
```diff
@@ -1,14 +1,23 @@
...
+
+
+      <div className="bg-light p-5 rounded-lg m-3">
+      <h1 className="display-4">Welcome to Loosey Goosey</h1>
+      <p className="lead">  Sips happen here </p>
+      <hr className="my-4"/>
+      <p> </p>
+    </div>
+      )
+      }
```
Comment (elysemontano) [inline app/javascript/components/pages/Home.js:19]: Indentation is still a little off for these.

File: app/javascript/components/pages/NotFound.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class NotFound extends Component {
+  render() {
+    return (
+    <div>
+    <h2>NotFound</h2>
```
Comment (elysemontano) [inline app/javascript/components/pages/NotFound.js:7]: Indentation

**Review by elysemontano — CHANGES_REQUESTED** [review]
Some more indentation and comments to address.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Two more comments that didn't get resolved from the last request change.

File: app/javascript/components/components/Header.js (hunk 3 above)
Comment (VMMarin) [reply app/javascript/components/components/Header.js:55]: We did test removing them, but everything broke. With the fragments, our code functions as it should.

Comment (elysemontano) [reply app/javascript/components/components/Header.js:54]: Sounds good!

**Review by elysemontano — APPROVED** [review]
Approved!

**Pull request merged**